tic-tac-toe/src/components/Game.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { calculateWinner } from "../helper";
+import Board from "./Board";
+
+const Game = () => {
+    const [history, setHistory] = useState([Array(9).fill(null)]);
+    const [stepNumber, setStepNumber] = useState(0);
+    const [xIsNext, setXisNext] = useState(true);
+    const winner = calculateWinner(history[stepNumber]);
+    const xO = xIsNext ? "x" : "o";
+
+    const handleClick = (i) => {
```

Comment:
A nice react hook for this is useCallback! since you're using some state elements in here (history and stepNumber), right now this will re-render the Game component any time one of these is updated. 

I think in the case of this project they're always called together and always trigger a re-render anyways but it's a good habit to get into for future projects.

[official useCallback documentation](https://reactjs.org/docs/hooks-reference.html#usecallback)
[useCallback documentation from a better source that explain why we use it](https://www.geeksforgeeks.org/react-js-usecallback-hook/#:~:text=useCallback%20will%20return%20a%20memoized,equality%20to%20prevent%20unnecessary%20renders.)